src/test/java/net/snowflake/client/jdbc/BindUploaderLatestIT.java

```diff
@@ -162,4 +163,20 @@ public void testSerializeCSVSimple() throws Exception {
     // After the uploader closes, it should clean up the CSV
     assertFalse(Files.exists(p));
   }
+
```

Comment:
Several tests already exist, but a new one has been added for a large batch size of bind parameters. It is about 10 times the size of the inputStream buffer.